lib/src/note/frequency.dart

```diff
@@ -0,0 +1,23 @@
+part of '../../music_notes.dart';
+
+/// Represents an absolute pitch, a physical frequency.
+class Frequency {
+  Frequency(this.hertz);
```

Comment:
Whenever possible, I would declare constant constructors alongside the `@immutable` class decoration:

```suggestion
@immutable
class Frequency {
  const Frequency(this.hertz);
```